libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp

```diff
@@ -7,21 +7,35 @@
 namespace eosio { namespace chain {
    static constexpr auto kv_payer_size = sizeof(account_name);
 
-   static uint32_t actual_value_size(const uint32_t raw_value_size) {
+   static uint32_t kv_rksdb_actual_value_size(const uint32_t raw_value_size) {
```

Comment:
Why were these functions renamed? The `kv_rksdb_` prefixes are a bit much and indicative that we should be namespacing these.  Was there clashing of function names?  Also, none of these functions are `inline`, meaning that they will cause duplicate symbols if we include in multiple compilation units.

Reply:
I thought the original names were too generic to be used outside kv_context_rocksdb. Will change to use name space and inline. Thanks.

Reply:
Can you revert them?  I renamed them in my branch.

Reply:
I renamed them and moved them, since I needed them as well.

Reply:
I am going to revert the naming changes, not introducing namespace for now so we can move forward quickly today. Will revisit this next week once everything is in.